test/test_common/certs_test_expected.cc

```diff
@@ -0,0 +1,47 @@
+#include <string>
```

Comment:
Seems these certificate data are from `test/common/ssl/test_data/`, which might be regenerated/modified in future.

You can just use `TestEnvironment::readFileToStringForTest` to read those data and use as expected strings.